src/platform/Darwin/dnssd/DnssdImpl.cpp

```diff
@@ -487,11 +487,10 @@ CHIP_ERROR ChipDnssdStopBrowse(intptr_t browseIdentifier)
 
     for (auto & resolve : resolves)
     {
-        resolve->Finalize(CHIP_ERROR_CANCELLED);
+        TEMPORARY_RETURN_IGNORED resolve->Finalize(CHIP_ERROR_CANCELLED);
     }
 
-    ctx->Finalize(CHIP_ERROR_CANCELLED);
-    return CHIP_NO_ERROR;
+    return ctx->Finalize(CHIP_ERROR_CANCELLED);
```

Comment:
Actually, no, this is wrong.  The semantics of Finalize is that it returns the error code it was passed.  This will make `ChipDnssdStopBrowse` _always_ return failure....

We probably want to change this Finalize behavior, but that's not something to do in this PR.